admin/spec/service/qr.service.spec.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+/* global describe, it, expect */
+
+const qrService = require('../../services/qr.service')
+const qrURL = require('../mocks/qr-url')
```

Comment:
@GeorgeChatzigiannis same issue re' camel. I won't add comments every time I see it =)

Reply:
also agree.  qrUrl please